BCH-Wunder-Project-team-4/Wunder.io
# Article Text Paragraph

## Link to ticket:

[Ticket](https://trello.com/c/6HLMNqyR)

## Changes proposed in this PR:

- Adds a new paragraph Article Body Text for Article content type. The formatted text field for this paragraph has extra styling for links (both internal and external), p, h2 & h3 element.
- Add more styling to article.tsx and rendering checks
- The width of main content is adjusted to be more easily readable
- (Minor change: Simple Quote label is now Quote)

## Screenshots

<img width="722" alt="Screenshot 2023-12-12 at 18 15 20" src="https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/122397061/4dda0bad-37a3-44bf-a30f-ca29d301e168">


## How to test:

Before: export local configs and changes and commit them

1. `git fetch origin article-text`
2. `git checkout article-text`
3. `lando drush cim`
4. Create an article with the new paragraph type
5. To see author picture, go to People in Drupal and add profile picture for admin user

## Best practices:

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
